common/maplib/map.mjs

```diff
@@ -152,6 +153,22 @@ class GameMap {
   getCellRenderInfo(coord, layer) {
     return this._maps.get(coord.mapName).getCellRenderInfo(coord, layer);
   }
+  /**
+   * Get Spawn Point List.
+   * @param {MapCoord} coord - The map coordinate.
+   * @return {list} spawn point - A list of the spawn point.
+   */
+  getSpawnPoint(coord){
```

Comment:
What is coord for? The list of spawn points should be unrelated to a coordinate. i.e. The list of spawn point should always be the same, there shouldn't be a coord input parameter that could change the list of spawn points.

Note that a coordinate is denoted by (world, x, y), whereby (x,y) could be the same for different world.